contract/report/report_contract.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" ?>
+<odoo>
+
+    <template id="report_contract_document">
+        <t t-call="report.html_container">
+            <t t-foreach="docs" t-as="o">
+                <t t-call="report.external_layout">
+                    <div class="page">
+                        <div class="oe_structure"/>
+                        <div class="row" id="partner_info">
+                            <div class="col-xs-5 col-xs-offset-7">
+                                <p id="partner_info"><strong>Partner:</strong></p>
+                                <div t-field="o.partner_id" t-field-options='{"widget": "contact", "fields": ["address", "name", "phone", "mobile", "fax", "email"], "no_marker": true, "phone_icons": true}'/>
+                                <p t-if="o.partner_id.vat">VAT: <span t-field="o.partner_id.vat"/></p>
+                            </div>
+                        </div>
+                        <div class="row" id="header_info">
+                            <div class="col-xs-3">
+                                <strong>Date Start: </strong><p t-field="o.date_start"/>
```

Comment:
This is inconsistent with your email, which has the data values encapsulated in `<strong>` vs the data keys as they are here. Please update one (I prefer no bold personally, but I don't use the stock reports and emails anyways so it doesn't affect me)